examples/eia/docs/index.md

```diff
@@ -37,24 +34,18 @@ const eiaBARef = FileAttachment("data/eia-bia-reference.csv").csv({typed: true})
 const us = await FileAttachment("data/us-states.json").json();
 const nation = us.features.find(({id}) => id === "nation");
 const statemesh = us.features.find(({id}) => id === "statemesh");
```

Comment:
The `await` here causes the entire block to wait for this file to load, rather than letting the promises resolve independently. One way to let them resolve independently is to make them all promises:

```suggestion
const us = FileAttachment("data/us-states.json").json();
const nation = us.then((us) => us.features.find(({id}) => id === "nation"));
const statemesh = us.then((us) => us.features.find(({id}) => id === "statemesh"));
```